ext/opcache/zend_accelerator_module.c

```diff
@@ -983,10 +1016,11 @@ ZEND_FUNCTION(opcache_compile_file)
 ZEND_FUNCTION(opcache_is_script_cached)
 {
 	zend_string *script_name;
+	bool file_cache = 0;
 
-	ZEND_PARSE_PARAMETERS_START(1, 1)
-		Z_PARAM_STR(script_name)
-	ZEND_PARSE_PARAMETERS_END();
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "S|b", &script_name, &file_cache) == FAILURE) {
+		RETURN_THROWS();
+	}
```

Comment:
I suppose the change away from fast-ZPP to regular ZPP is due to not knowing how this works instead of a deliberate change? (see https://wiki.php.net/rfc/fast_zpp)

```suggestion
	ZEND_PARSE_PARAMETERS_START(1, 2)
		Z_PARAM_STR(script_name)
		Z_PARAM_OPTIONAL
		Z_PARAM_BOOL(file_cache)
	ZEND_PARSE_PARAMETERS_END();
```

Reply:
Thank you for the help with this!